src/test/java/org/flickit/flickitassessmentcore/application/service/evidence/DeleteEvidenceServiceTest.java

```diff
@@ -0,0 +1,59 @@
+package org.flickit.flickitassessmentcore.application.service.evidence;
+
+import org.flickit.flickitassessmentcore.application.port.in.evidence.DeleteEvidenceUseCase;
+import org.flickit.flickitassessmentcore.application.port.out.evidence.CheckEvidenceExistencePort;
+import org.flickit.flickitassessmentcore.application.port.out.evidence.DeleteEvidencePort;
+import org.flickit.flickitassessmentcore.application.service.exception.ResourceNotFoundException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.flickit.flickitassessmentcore.common.ErrorMessageKey.DELETE_EVIDENCE_EVIDENCE_NOT_FOUND;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class DeleteEvidenceServiceTest {
+
+    @InjectMocks
+    private DeleteEvidenceService service;
+    @Mock
+    private DeleteEvidencePort deleteEvidencePort;
+    @Mock
+    private CheckEvidenceExistencePort checkEvidenceExistencePort;
+
+    @Test
+    void deleteEvidence_IdGiven_Delete() {
```

Comment:
add `test` at beginning of test method name

do same for other tests of this class